avm/res/azure-stack-hci/cluster/tests/e2e/2nodeswitched.defaults/main.test.bicep

```diff
@@ -0,0 +1,304 @@
+  scope: resourceGroup
+  params: {
+    clusterNodeNames: clusterNodeNames
+    clusterWitnessStorageAccountName: clusterWitnessStorageAccountName
+    deployArcGateway: deployArcGateway
+    deploymentPrefix: deploymentPrefix
+    deploymentUsername: deploymentUsername
+    deploymentUserPassword: localAdminAndDeploymentUserPass
+    domainOUPath: domainOUPath
+    hciResourceProviderObjectId: hciResourceProviderObjectId
+    hciHostAssignPublicIp: hciHostAssignPublicIp
+    keyVaultName: keyVaultName
+    keyVaultDiagnosticStorageAccountName: keyVaultDiagnosticStorageAccountName
+    localAdminPassword: localAdminAndDeploymentUserPass
+    localAdminUsername: localAdminUsername
+    location: enforcedLocation
+    arbDeploymentAppId: arbDeploymentAppId
+    arbDeploymentSPObjectId: arbDeploymentSPObjectId
+    arbDeploymentServicePrincipalSecret: arbDeploymentServicePrincipalSecret
+    vnetSubnetId: vnetSubnetId
+    hciNodeCount: length(clusterNodeNames)
+    switchlessStorageConfig: false
+    hciISODownloadURL: hciISODownloadURL
+    hciVHDXDownloadURL: hciVHDXDownloadURL
+  }
+}
+
+module cluster_validate '../../../main.bicep' = {
+  dependsOn: [
+    hciDependencies
+  ]
+  name: '${uniqueString(deployment().name, enforcedLocation)}-test-clustervalidate-${serviceShort}'
+  scope: resourceGroup
+  params: {
+    name: name
+    customLocationName: customLocationName
+    clusterNodeNames: clusterNodeNames
+    clusterWitnessStorageAccountName: clusterWitnessStorageAccountName
+    defaultGateway: defaultGateway
+    deploymentMode: 'Validate'
+    deploymentPrefix: deploymentPrefix
+    dnsServers: dnsServers
+    domainFqdn: domainFqdn
+    domainOUPath: domainOUPath
+    endingIPAddress: endingIPAddress
+    enableStorageAutoIp: enableStorageAutoIp
+    keyVaultName: keyVaultName
+    networkIntents: networkIntents
+    startingIPAddress: startingIPAddress
+    storageConnectivitySwitchless: false
+    storageNetworks: storageNetworks
+    subnetMask: subnetMask
+  }
+}
+
+module testDeployment '../../../main.bicep' = {
+  dependsOn: [
+    hciDependencies
+    cluster_validate
+  ]
+  name: '${uniqueString(deployment().name, enforcedLocation)}-test-clusterdeploy-${serviceShort}'
+  scope: resourceGroup
+  params: {
+    name: name
+    clusterNodeNames: clusterNodeNames
+    clusterWitnessStorageAccountName: clusterWitnessStorageAccountName
+    customLocationName: customLocationName
+    defaultGateway: defaultGateway
+    deploymentMode: 'Deploy'
+    deploymentPrefix: deploymentPrefix
+    dnsServers: dnsServers
+    domainFqdn: domainFqdn
+    domainOUPath: domainOUPath
+    endingIPAddress: endingIPAddress
+    enableStorageAutoIp: enableStorageAutoIp
+    keyVaultName: keyVaultName
+    networkIntents: networkIntents
+    startingIPAddress: startingIPAddress
+    storageConnectivitySwitchless: false
+    storageNetworks: storageNetworks
+    subnetMask: subnetMask
+  }
+}
+
+type networkIntent = {
```

Comment:
This file should not have any types as no input parameters that would need types should be passed into this file

Reply:
Removed types from e2e testing files

Reply:
Hey @mbrat2005, I think you may have commited, but not pushed the changes as they don't show up in GitHub yet